### ISSUE TITLE
cadastrapp toolbar isnt responsive/mobile-compatible

### ISSUE THREAD

**Issue by landryb** [georchestra/mapstore2-cadastrapp#85]
we need to find a way to adapt the toolbar width to the screen, in the mobile view buttons arent even displayed.

![image](https://user-images.githubusercontent.com/3421760/108839277-27bb4400-75d5-11eb-8a78-78aa6dc9281f.png)
![image](https://user-images.githubusercontent.com/3421760/108839340-3f92c800-75d5-11eb-88cd-63d46a1d63a4.png)

in a tablet view it's "a bit better" but still..

![image](https://user-images.githubusercontent.com/3421760/108839444-5fc28700-75d5-11eb-9b2d-a1dc15575172.png)
![image](https://user-images.githubusercontent.com/3421760/108839520-779a0b00-75d5-11eb-92ce-7c63687e0cfa.png)

maybe the toolbar could adapt its layout/width/orientation depending on the screen orientation ?

**Comment by tdipisa** [comment]
@landryb as we agreed during one of the first meeting, the mobile compliance is not a requirement for the cadastrapp tool, therefore we didn't concentrate to much effort on this. I think this can be considered as a future enhancement, if really needed.

**Comment by landryb** [comment]
yeah i remember it wasnt a requirement, sure, it's an enhancement, and i'm pretty sure at some point it will be needed ;) Just filing so that its not lost..

**Comment by MaelREBOUX** [comment]
> the mobile compliance is not a requirement for the cadastrapp tool

I confirm. We said that the most seen use case in office use.
And we did not choose yet between adapative code or a dedicated mobile plugin with lighter functionalities.

To discuss later...
